spec/features/admin_views_recent_payments_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature 'Admin views recent payment', {js: true, vcr: true} do
+  given(:admin) { Fabricate(:admin) }
+  given(:joe) { Fabricate(:user, fullname: 'Joe Doe', email: 'joe@example.com') }
+
+  before do
+    Fabricate(:payment, user: joe, reference_id: "asdfg")
+  end
+
+  scenario "admin sees the 10 recent payments" do
```

Comment:
This one is as well. It should be something more like "admin sees recent payments". You are not verifying 10.